### ISSUE TITLE
[bugs]: the pick-focus and pick-close binds don't work

### ISSUE THREAD

**Issue by justchokingaround** [willothy/nvim-cokeline#204]
![image](https://github.com/willothy/nvim-cokeline/assets/44473782/0c58604f-ab35-4a75-97f6-ed7a3d74aa3a)
as the title says, these binds don't work (after i run that command, nothing happens). for comparison they work fine in nvim-bufferline:
![image](https://github.com/willothy/nvim-cokeline/assets/44473782/e2168627-7592-4871-8060-b6cc835074b1)

**Comment by willothy** [comment]
Could you please provide a minimal init file that can reproduce this issue, and some info about your Neovim version? This works totally fine for me.

**Comment by justchokingaround** [comment]
sure, i'll try doing that later, maybe tomorrow or in two days. i was testing with neovim v0.10.0 tho, so that might be why

**Comment by willothy** [comment]
I'm on v0.10.0 as well. I'm not using the plug mappings though, maybe that's the difference. I'll look into it a bit more.

No rush, I'm super busy with work rn so I don't have a ton of time for open-source rn. I'll try to get to this ASAP once you have some more info though :)

**Comment by justchokingaround** [comment]
yeah sure np. just fyi i tried the same thing with lua and it had the same result

**Comment by willothy** [comment]
Hmm alright, that's good to know. Thanks!

**Comment by justchokingaround** [comment]
```lua
local lazypath = vim.fn.stdpath 'data' .. '/lazy/lazy.nvim'
if not vim.loop.fs_stat(lazypath) then
	local lazyrepo = 'https://github.com/folke/lazy.nvim.git'
	vim.fn.system { 'git', 'clone', '--filter=blob:none', '--branch=stable', lazyrepo, lazypath }
end ---@diagnostic disable-next-line: undefined-field
vim.opt.rtp:prepend(lazypath)

require('lazy').setup({
	{
		"willothy/nvim-cokeline",
		dependencies = {
			"nvim-lua/plenary.nvim", -- Required for v0.4.0+
			"nvim-tree/nvim-web-devicons", -- If you want devicons
			"stevearc/resession.nvim" -- Optional, for persistent history
		},
		config = true
	},
	{ "nvim-telescope/telescope.nvim" },
})
vim.api.nvim_set_keymap('n', '<space><space>', ':Telescope find_files<CR>', { noremap = true, silent = true })
```

here's a minimal config u can run with `nvim --clean -u temp.lua` . the only keybinds which didn't work in my testing were `<Plug>(cokeline-pick-focus)` and `<Plug>(cokeline-pick-close)`

**Comment by willothy** [comment]
Thanks! I'll look into this as soon as I can.
